Makefile

```diff
@@ -94,9 +94,15 @@ clean:
 	rm -rf android/app/build
 
 post-install:
+	@cd ./node_modules/remotedev-server
+	@npm install socketcluster@5.0.4
```

Comment:
Why not include this in the package.json? Also, we should use yarn here if possible since that's what we use elsewhere

Reply:
Yep, good catch.